pl.lproj/Localizable.strings

```diff
@@ -82,18 +82,18 @@
 
 "COLLECTION_VIEW_EMPTY_SECTION_TITLE" = "Brak Stacji";
 
-"WELCOME_COPY" = "Witamy w Broadcasts! Dodaliśmy już parę naszych Irlandzkich stacji radiowych, aby twoja biblioteka nie była pusta.\n\nJeżeli wolisz, możesz dodać ręcznie własne stacje lub wybrać którąś z tysiąca stacji w naszym katalogu.";
+"WELCOME_COPY" = "Witamy w Broadcasts! Dodaliśmy już kilka irlandzkich stacji radiowych, aby Twoja biblioteka nie była pusta.\n\nJeżeli wolisz, możesz dodać ręcznie własne stacje lub wybrać którąś z tysiąca stacji z naszego katalogu.";
 
-"WELCOME_UPSELL" = "Witamy w Broadcasts! W tej darmowej wersji dodaliśmy już parę naszych Irlandzkich stacji radiowych, aby biblioteka nie była pusta.\n\nJeżeli przejdziesz do płatnej wersji, możliwe będzie dodanie ręcznie własnych stacji lub wybranie którejś z tysiąca stacji w naszym katalogu.";
+"WELCOME_UPSELL" = "Witamy w Broadcasts! W tej darmowej wersji dodaliśmy już kilka irlandzkich stacji radiowych, aby Twoja biblioteka nie była pusta.\n\nJeżeli przejdziesz do płatnej wersji, możliwe będzie dodanie ręcznie własnych stacji lub wybranie którejś z tysiąca stacji z naszego katalogu.";
 
 "WELCOME_ADD_PRESETS" = "Dodaj Stacje Początkowe";
 "WELCOME_NO_THANKS" = "Nie, dziękuję";
 
 "GENERATED_COLLECTION_TITLE" = "Moja Kolekcja";
 
-"UPSELL_COPY" = "Dzięki za korzystanie z Broadcasts!\n\nJeżeli chcesz, możesz odblokować zaawansowane funkcje zakupem w aplikacji.\n\nJeżeli wolisz, możesz po prostu edytować lub korzystać dalej ze stacji początkowych!";
+"UPSELL_COPY" = "Dzięki za korzystanie z Broadcasts!\n\nJeżeli chcesz, możesz odblokować zaawansowane funkcje za pomocą mikropłatności w aplikacji.\n\nJeżeli wolisz, możesz po prostu edytować lub korzystać dalej ze stacji początkowych!";
```

Comment:
Read the original, so submitting further changes, to make it sound as close as possible:

`"UPSELL_COPY" = "Dzięki za korzystanie z Broadcasts!\n\nJeżeli chcesz, możesz odblokować zaawansowane funkcje za pomocą mikropłatności w aplikacji.\n\nJeżeli wolisz, możesz po prostu edytować lub korzystać dalej ze stacji początkowych!";`

→

`"UPSELL_COPY" = "Dziękuję za korzystanie z Broadcasts!\n\nJeżeli chcesz, możesz odblokować zaawansowane funkcje za pomocą mikropłatności w aplikacji.\n\nA jeśli wolisz, możesz również dalej korzystać z domyślnych stacji i je edytować!";`

Reply:
I think “dzięki” would be the right option as it means thanks rather than thank you.

Reply:
I changed it to dziękuję because dzięki is much less formal in Polish than thanks in English. Either will be fine though.

Reply:
I fixed the capitalisation, implemented your changes and a few more. Thank you once again for pointing out the errors!